katdal/chunkstore_rados.py

```diff
@@ -86,8 +89,13 @@ def from_config(cls, config, pool, keyring=None, timeout=5.):
         try:
             if isinstance(config, dict):
                 cluster = rados.Rados(conf=config)
-            else:
+            elif os.path.isfile(config):
                 cluster = rados.Rados(conffile=config)
+            else:
+                with tempfile.NamedTemporaryFile() as f:
+                    f.write(config)
+                    f.seek(0)
```

Comment:
What is this for?

Reply:
I assumed the file pointer would be at the end of the file... Enlighten me :-)

Reply:
It doesn't matter where the pointer associated with `f` is, because you're not passing `f` anywhere, you're only passing the filename, and whatever opens it will have its own file pointer.

However, I think in some implementations, seeking may also flush the file to the OS. You should probably have an explicit `f.flush()`.

Reply:
Good pointer... Forgot about that! Will add the flush.

Reply:
The `f.flush()` was necessary, BTW. Good reminder.